apps/next/.env.example

```diff
@@ -22,4 +22,6 @@ POSTGRES_CA_DATA=x
 POSTGRES_HOST=x
 POSTGRES_USER=x
 POSTGRES_PASSWORD=x
-POSTGRES_DATABASE=x
\ No newline at end of file
+POSTGRES_DATABASE=x
+# Set this to true to disable the confirmation step during `drizzle-kit push:pg`
+DRIZZLE_DISABLE_STRICT_PUSHES=
```

Comment:
could the same be accomplished by checking if the CI env variable is set to 'true'? https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
Done.